magnolia-voting-app/src/main/resources/voting-contentApp/templates/components/poll.ftl

```diff
@@ -0,0 +1,56 @@
+[#assign pollSelector = state.getSelector()!]
```

Comment:
Since, there's no defined style, I wonder if result will be same as mockup.

Reply:
I have styled the template like the mockup.